webui/react/src/pages/WorkspaceDetails/WorkspaceProjects.tsx

```diff
@@ -176,6 +175,8 @@ const WorkspaceProjects: React.FC<Props> = ({ workspace, id, pageRef }) => {
   }, []);
 
   const columns = useMemo(() => {
+    if (Loadable.isLoading(users)) return [];
```

Comment:
this is doing thing where it returns empty columns, which i think is what was causing the page crash on the experiment list before